[WIP] RHDEVDOCS 3910 and RHDEVDOCS-3902- Service binding discoverability and Topology usability improvements (#47760)
openshift/openshift-docs: modules/odc-discovering-and-identifying-operator-backed-bindable-services.adoc

:_content-type: PROCEDURE
[id="odc-discovering-and-identifying-operator-backed-bindable-services_{context}"]
== Discovering and identifying Operator-backed bindable services

As an operator provider or a user who creates backing services, you must know which services are bindable before you create a backing service. The *Developer* perspective helps you discover and identify such bindable services. 

.Procedure

> To discover and identify Operator-backed bindable services, consider the following alternative approaches:
> 
> * Click *+Add* -> *Developer Catalog* -> *Operator Backed* to see the Operator-backed tiles. Operator-backed services that support service binding features have a *Bindable* badge on the tiles.
> * On the left pane of the *Operator Backed* page, select the *Bindable* checkbox.
> ** Optional: Click on the help icon next to *Service binding* to see more information about bindable services.
> * Click  *+Add* -> *Add* and search for Operator-backed services. When you click on the service that is bindable, you can view the *Bindable* badge in the side panel to the right.

Reviewer: ```suggestion

* To discover and identify Operator-backed bindable services, consider the following alternative approaches:
+
* Click *+Add* -> *Developer Catalog* -> *Operator Backed* to see the Operator-backed tiles. Operator-backed services that support service binding features have a *Bindable* badge on the tiles.
* On the left pane of the *Operator Backed* page, select the *Bindable* checkbox.
** Optional: Click the help icon next to *Service binding* to see more information about bindable services.
* Click  *+Add* -> *Add* and search for Operator-backed services. When you click the service that is bindable, you can view the *Bindable* badge in the side panel to the right.
```

Reviewer: Or consider converting these to conditional steps, as shown below. https://github.com/openshift/openshift-docs/pull/47760#discussion_r921122694

Author: Hello, @rolfedh 
Thanks for all the suggestions. They are very helpful. I will be incorporating all of them except this feedback. I have tried a different approach. Also, regarding line 12 in your comment, I am retaining my version to keep the technical accuracy intact.

> * Click  *+Add* -> *Add* and search for Operator-backed services. When you click on the service that is bindable, you can view the *Bindable* badge in the side panel to the right.

Reviewer: ```suggestion
* Click  *+Add* -> *Add* and search for Operator-backed services. When you click the bindable service, you can view the *Bindable* badge in the side panel to the right.
```

Author: Done.

:_content-type: PROCEDURE
[id="odc-discovering-and-identifying-operator-backed-bindable-services_{context}"]
== Discovering and identifying Operator-backed bindable services

> As an operator provider or a user who creates backing services, you must know which services are bindable before you create a backing service. The *Developer* perspective helps you discover and identify such bindable services. 

Reviewer 2: ```suggestion
As a user if you want to create a bindable service, you must know which services are bindable. The *Developer* perspective helps you discover and identify such bindable services. 
```

Reviewer 2: If you want to highlight what are bindable services you can use this - `Bindable services are services that can be easily consumed by applications because they expose their binding data (credentials, connection details, volume mounts, secrets, etc.) in a standard way.`

Reviewer 2: Can you keep the above text as is because that is more technically correct?

Author: @divyanshiGupta Could you please confirm now?
![image](https://user-images.githubusercontent.com/89447174/179499996-070535ce-04d1-4fca-9156-b027cd16511c.png)

Thank you!

> As a user, if you want to create a bindable service, you must know which services are bindable. Bindable services are services that can be easily consumed by applications because they expose their binding data such as credentials, connection details, volume mounts, secrets, and other binding data in a standard way. The *Developer* perspective helps you discover and identify such bindable services.

Reviewer 3: ```suggestion
As a user, if you want to create a bindable service, you must know which services are bindable. Bindable services are services that the applications can consume easily because they expose their binding data such as credentials, connection details, volume mounts, secrets, and other binding data in a standard way. The *Developer* perspective helps you discover and identify such bindable services.
```

Author: Done.

Author: Hello @Srivaralakshmi. I have incorporated your suggestion. Could you please confirm? Thank you!
![image](https://user-images.githubusercontent.com/89447174/179697999-9406e5e4-923b-47ee-9b70-7203ef512f3d.png)
Cc @divyanshiGupta

.Procedure
* To discover and identify Operator-backed bindable services, consider the following alternative approaches:
+
** Click *+Add* -> *Developer Catalog* -> *Operator Backed* to see the Operator-backed tiles. Operator-backed services that support service binding features have a *Bindable* badge on the tiles.
** On the left pane of the *Operator Backed* page, select the *Bindable* checkbox.
+
[TIP]
====
Click the help icon next to *Service binding* to see more information about bindable services.
====

> +

Author: Done.
